include: install header files without prefix dir inih

The other flavor of inih which adapted by Fedora and Ubuntu installs
header files to /usr/include directly. And xfsprogs 5.10.0 also checks
ini.h without extra search path and fails. So install header files
without prefix dir inih.

Upstream-Status: Submitted [https://github.com/OSSystems/inih/pull/4]

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 include/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/CMakeLists.txt b/include/CMakeLists.txt
index 7a46ee7..4a9033f 100644
--- a/include/CMakeLists.txt
+++ b/include/CMakeLists.txt
@@ -1 +1 @@
-install(FILES ini.h INIReader.h DESTINATION ${CMAKE_INSTALL_PREFIX}/include/inih)
+install(FILES ini.h INIReader.h DESTINATION ${CMAKE_INSTALL_PREFIX}/include)
-- 
2.17.1

